web/js/palettes/legend.js

```diff
@@ -59,6 +59,7 @@ export function palettesLegend(spec) {
   };
 
   var renderScale = function ($legendPanel, legend, index, layerId) {
+    var $container, $colorbar;
```

Comment:
You can add the `var` keyword to the next two lines where these variables are defined, instead of breaking the variable definitions into a separate line like this.